core/components/css-utilities/Miscellaneous/Miscellaneous.story.tsx

```diff
@@ -24,20 +24,631 @@ export const miscellaneous = () => {
       className: 'cursor-pointer',
       properties: 'cursor: pointer;',
     },
+    {
+      className: 'white-space-nowrap',
+      properties: 'white-space: nowrap;',
+    },
+    {
+      className: 'bottom-0',
+      properties: 'bottom: 0;',
+    },
+    // Visibility utilities
+    {
+      className: 'visible',
+      properties: 'visibility: visible;',
+    },
+    {
+      className: 'invisible',
+      properties: 'visibility: hidden;',
+    },
+    // User select utilities
+    {
+      className: 'user-select-none',
+      properties: 'user-select: none;',
+    },
```

Comment:
Please create a separate table for each group of class for better user discoverability and avoid pagination in tables to display different groups of classes